specification/keyvault/data-plane/Microsoft.KeyVault/2016-10-01/keyvault.json

```diff
@@ -1252,22 +1259,23 @@
           "DeletedSecrets"
         ],
         "operationId": "RecoverDeletedSecret",
-        "description": "Recovers the deleted secret back to its current version under /secrets. Authorization: requires the secrets/recover permission.",
+        "summary": "Recovers the deleted secret to the latest version.",
+        "description": "Recovers the deleted secret in the specified vault. It requires the secrets/recover permission.",
```

Comment:
>Recovers the deleted secret in the specified vault. It requires the secrets/recover permission. [](start = 24, length = 95)

should this mention that it can only be performed on a soft delete enabled vault?